### PR TITLE
Check the language driver version

### PR DESCRIPTION
**Pull request by vmarkovtsev**
Signed-off-by: Vadim Markovtsev <vadim@sourced.tech>

This feature requires an update of `lookout-sdk-ml`

### CODE REVIEW

File: lookout/style/format/langs/javascript/__init__.py [hunk 1]
```diff
@@ -1 +1,2 @@
 """Javascript specific resources and methods for the format analyzer."""
+__version__ = "==1.3.0"
```
Comment (zurk) [inline lookout/style/format/langs/javascript/__init__.py]:
```suggestion
__version__ = "1.3.0"  # Required bblfsh driver version
```

File: lookout/style/format/analyzer.py [hunk 2]
```diff
@@ -553,3 +560,19 @@ def _load_config(cls, config: Mapping[str, Any]) -> Mapping[str, Any]:
...
+        Return the value indicating whether the Babelfish driver version for the specified \
+        language is supported.
+        """
+        language = language.lower()
+        try:
+            data_service.check_bblfsh_driver_versions([
+                language + import_module("lookout.style.format.langs.%s" % language).__version__])
+            return True
+        except UnsatisfiedDriverVersionError as e:
+            log.warning("language %s was skipped, Babelfish driver is incompatible: %s", e)
```
Comment (zurk) [inline lookout/style/format/analyzer.py]: I think it is an error.

File: lookout/style/format/analyzer.py [hunk 3]
```diff
@@ -553,3 +560,19 @@ def _load_config(cls, config: Mapping[str, Any]) -> Mapping[str, Any]:
...
+    def _check_language_version(cls, language: str, data_service: DataService,
+                                log: logging.Logger) -> bool:
+        """
+        Return the value indicating whether the Babelfish driver version for the specified \
+        language is supported.
+        """
+        language = language.lower()
+        try:
+            data_service.check_bblfsh_driver_versions([
+                language + import_module("lookout.style.format.langs.%s" % language).__version__])
```
Comment (zurk) [inline lookout/style/format/analyzer.py]:
```suggestion
                "%s==%s" % (language, import_module("lookout.style.format.langs.%s" % language).__version__]))
```

File: lookout/style/format/langs/javascript/__init__.py (hunk 1 above)
Comment (vmarkovtsev) [reply lookout/style/format/langs/javascript/__init__.py]: Nope

File: lookout/style/format/analyzer.py (hunk 3 above)
Comment (vmarkovtsev) [reply lookout/style/format/analyzer.py]: Nope. It supports any valid version specifier compatible with PEP (don't remember the number, you can google yourself).

**Comment by vmarkovtsev** [comment]
@irinakhismatullina  I am going to hack the dependency hell here.

**Comment by irinakhismatullina** [comment]
Okey, i will rebase after this one is merged

**Comment by vmarkovtsev** [comment]
@zurk `test_evaluate_smoke_entry_integration` is failing here, can you please help

**Comment by zurk** [comment]
here: https://github.com/src-d/lookout-sdk-ml/pull/86

**Pull request merged**